tests/template_tests/filter_tests/test_stringformat.py

```diff
@@ -29,6 +29,13 @@ class FunctionTests(SimpleTestCase):
 
     def test_format(self):
         self.assertEqual(stringformat(1, '03d'), '001')
+        self.assertEqual(stringformat((1, 2, 3), 's'), '(1, 2, 3)')
+        self.assertEqual(stringformat([1, None], 's'), '[1, None]')
+        self.assertEqual(stringformat((1,), 's'), '(1,)')
+        self.assertEqual(stringformat({1, 2}, 's'), '{1, 2}')
+        self.assertEqual(stringformat({1: 2, 2: 3}, 's'), '{1: 2, 2: 3}')
 
     def test_invalid(self):
         self.assertEqual(stringformat(1, 'z'), '')
+        self.assertEqual(stringformat(object(), 'd'), '')
+        self.assertEqual(stringformat(None, 'd'), '')
```

Comment:
You've added seven asserts, but only two are related with this fix, so I think it will be better to send:
```python
self.assertEqual(stringformat([1, None], 's'), '[1, None]')
self.assertEqual(stringformat({1, 2}, 's'), '{1, 2}')
self.assertEqual(stringformat({1: 2, 2: 3}, 's'), '{1: 2, 2: 3}')
...
self.assertEqual(stringformat(object(), 'd'), '')
self.assertEqual(stringformat(None, 'd'), '')
```
in advance in a separate PR.